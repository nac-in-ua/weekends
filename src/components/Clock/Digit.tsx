type DigitProps = {
  digit: number
}

function Digit({ digit }: DigitProps) {
  return (
    <div
      data-testid="digit"
      className="flex h-20 items-center justify-center text-5xl font-thin"
    >
      {digit}
    </div>
  )
}

export default Digit
