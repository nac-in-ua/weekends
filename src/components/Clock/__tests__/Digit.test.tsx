import Digit from '../Digit'
import { render, screen } from '@testing-library/react'

describe('Digit', () => {
  it('should render digit', () => {
    render(<Digit digit={12} />)
    const title = screen.getByText(/12/i)
    expect(title).toBeInTheDocument()
    expect(title).toMatchSnapshot()
  })
})
