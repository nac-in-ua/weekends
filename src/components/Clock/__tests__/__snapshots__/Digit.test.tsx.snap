// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Digit should render digit 1`] = `
<div
  class="flex h-20 items-center justify-center text-5xl font-thin"
  data-testid="digit"
>
  12
</div>
`;
